site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -307,6 +307,64 @@ a ClusterQueue can borrow up to the sum of nominal quotas from all the
 ClusterQueues in the cohort. So for the yamls listed above, `team-b-cq` can
 borrow `12+9` CPUs.
 
+### LendingLimit
+
+To limit the amount of resources that a ClusterQueue can lend to others,
+you can set the `.spec.resourcesGroup[*].flavors[*].resource[*].lendingLimit`
+[quantity](https://kubernetes.io/docs/reference/kubernetes-api/common-definitions/quantity/) field.
+
+#### Enabling feature LendingLimit
+
+LendingLimit is an `Alpha` feature disabled by default, check the [Change the feature gates configuration](/docs/installation/#change-the-feature-gates-configuration) section of the [Installation](/docs/installation/) for details.
+
+> _Available in Kueue v0.6.0 and later_
+
+As an example, assume you created the following two ClusterQueues:
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "team-a-cq"
+spec:
+  namespaceSelector: {} # match all.
+  cohort: "team-ab"
+  resourceGroups:
+  - coveredResources: ["cpu"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "cpu"
+        nominalQuota: 9
+```
+
+```yaml
+apiVersion: kueue.x-k8s.io/v1beta1
+kind: ClusterQueue
+metadata:
+  name: "team-b-cq"
+spec:
+  namespaceSelector: {} # match all.
+  cohort: "team-ab"
+  resourceGroups:
+  - coveredResources: ["cpu"]
+    flavors:
+    - name: "default-flavor"
+      resources:
+      - name: "cpu"
+        nominalQuota: 12
+        lendingLimit: 1
+```
+
+In this case, because we set lendingLimit in ClusterQueue `team-b-cq`, if 
+ClusterQueue `team-b-cq` has no admitted Workloads, then ClusterQueue `team-a-cq` 
+can admit Workloads with resources adding up to `9+1=10` CPUs. 
```

Comment:
```suggestion
In this case, because we set the lendingLimit in ClusterQueue `team-b-cq`, 
then ClusterQueue `team-a-cq` can at most admit Workloads with quota adding 
up to `9+1=10` CPUs. 
```